scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -184,8 +184,10 @@ trait Config extends Serializable {
    * with a class to serialize to bootstrap the process:
    * Left((classOf[serialization.KryoHadoop], myInstance))
    */
-  def setSerialization(kryo: Either[(Class[_ <: KryoInstantiator], KryoInstantiator), Class[_ <: KryoInstantiator]],
-    userHadoop: Seq[Class[_ <: HSerialization[_]]] = Nil): Config = {
+  def setSerialization(
+    kryo: Either[(Class[_ <: KryoInstantiator], KryoInstantiator), Class[_ <: KryoInstantiator]],
+    userHadoop: Seq[Class[_ <: HSerialization[_]]] = Nil,
+    additionalSerializedClasses: Set[Class[_]] = Set.empty): Config = {
```

Comment:
can we remove the `additionalSerializedClasses` here? I think callers should just do:
```scala
.setSerialization( ... )
.addCascadingClassSerializationTokens(additionalSerializedClasses)
```

But another concern is that sadly both cascading and kryo need to give tokens to classes. Can we make a kryo registrar:
https://github.com/twitter/chill/blob/develop/chill-java/src/main/java/com/twitter/chill/IKryoRegistrar.java

that registers any classes not already added? Then it can be combined with the KryoInstantiator:
https://github.com/twitter/chill/blob/develop/chill-java/src/main/java/com/twitter/chill/KryoInstantiator.java#L98